tools/pkg/build.py

```diff
@@ -748,6 +748,10 @@ def errfn(fn, path, err):
         ctx.info(f"Copying '{src.relative_to(tools.utils.REPO_ROOT)}' to '{dst}' ...")
         shutil.copyfile(src, dst)
 
+    # Add package type file for package grain
+    with open(pathlib.Path(site_packages) / "salt" / "_pkg.txt", "w") as fp:
+        fp.write("onedir")
```

Comment:
```suggestion
    pathlib.Path(site_packages).joinpath("salt", "_pkg.txt").write_text("onedir")
```
This is just an FYI, not a change request.